banzai/calibrations.py

```diff
@@ -225,8 +236,10 @@ def create_master_calibration_header(old_header, images):
     return header
 
 
-def run_master_maker(image_path_list, runtime_context, frame_type):
-    images = [image_utils.read_image(image_path, runtime_context) for image_path in image_path_list]
+def run_master_maker(calibration_image_records, runtime_context, frame_type):
+    file_info_list= [{'path': os.path.join(record.filepath, record.filename),
+                      'frameid': record.frameid} for record in calibration_image_records]
+    images = [image_utils.read_image(file_info, runtime_context) for file_info in file_info_list]
```

Comment:
The caching logic may be even better to have in here.

Reply:
See comment above.